test/sim/moves/metalburst.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Metal Burst', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it('should run conditions for submove', function () {
+		battle = common.createBattle({seed: [1, 2, 3, 4]}); // Snorlax needs to sleep for more than 1 turn
+		battle.setPlayer('p1', {team: [{species: 'snorlax', moves: ['sleeptalk', 'metalburst']}]});
+		battle.setPlayer('p2', {team: [{species: 'breloom', moves: ['spore', 'sonicboom']}]});
+		battle.makeChoices('move metalburst', 'move sonicboom');
+		const breloomHpTurn1 = battle.p2.active[0].hp;
+		assert.equal(breloomHpTurn1, battle.p2.active[0].maxhp - battle.dex.getMove('sonicboom').damage * 1.5);
+		battle.makeChoices('move sleeptalk', 'move spore');
+		battle.makeChoices('move sleeptalk', 'move sonicboom');
+		assert.equal(battle.p2.active[0].hp, breloomHpTurn1 - battle.dex.getMove('sonicboom').damage * 1.5);
+	});
+
+	it('should target the opposing Pokemon that hit the user with an attack most recently that turn', function () {
+		// The seed should select venusaur if the test would otherwise fail
+		battle = common.createBattle({gameType: 'doubles', seed: [3, 4, 5, 6]});
```

Comment:
This shouldn't need an RNG seed. Metal Burst doesn't randomly target.

Reply:
You're right, it shouldn't. This seed is so that if the code is _incorrect_, the test will actually fail instead of passing as a false negative.

Reply:
Gotcha; if it _were_ random, it _would_ target Venusaur. Looks fine then.